charts/tidb-operator/templates/scheduler-rbac.yaml

```diff
@@ -51,7 +51,11 @@ rules:
   verbs: ["create"]
 - apiGroups: ["coordination.k8s.io"]
   resources: ["leases"]
+  {{- if eq .Values.appendReleaseSuffix true}}
```

Comment:
Code here https://github.com/pingcap/tidb-operator/blob/v1.2.0-beta.1/charts/tidb-operator/templates/scheduler-rbac.yaml#L118 also needs to update.